src/ControlSystem/Actions/Initialization.hpp

```diff
@@ -58,6 +69,32 @@ struct InitializeControlSystem {
                     const Parallel::GlobalCache<Metavariables>& /*cache*/,
                     const ArrayIndex& /*array_index*/, ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) {
+    // Move all the control system inputs into their own tags in the databox so
+    // we can use them easily later
+    auto control_sys_inputs =
+        db::get_mutable_reference<
+            control_system::Tags::ControlSystemInputs<ControlSystem>>(
+            make_not_null(&box))
+            .get_inputs();
+    ::Initialization::mutate_assign<tags_to_be_initialized>(
+        make_not_null(&box),
+        std::move(tuples::get<control_system::Tags::Averager<deriv_order>>(
+            control_sys_inputs)),
+        std::move(tuples::get<control_system::Tags::Controller<deriv_order>>(
+            control_sys_inputs)),
+        std::move(tuples::get<control_system::Tags::TimescaleTuner>(
+            control_sys_inputs)));
+
+    // Set the initial time between updates using the initial timescale
+    const auto& tuner = db::get<control_system::Tags::TimescaleTuner>(box);
+    const double current_min_timescale = min(tuner.current_timescale());
```

Comment:
Maybe not for this PR, but we will have to guarantee somehow that there are no control system updates attempted before the last time used during integrator self-start.  The only idea I've had is tying the first update to the slab size, but that might be confusing because that's probably not an expected effect of changing the slab size.